colorpickerview/src/main/java/com/skydoves/colorpickerview/ColorPickerView.java

```diff
@@ -109,7 +109,9 @@ public ColorPickerView(Context context, AttributeSet attrs) {
     getAttrs(attrs);
     onCreate();
   }
-
+  public ImageView getSelector(){
+    return this.selector;
+  }
```

Comment:
Can we move this function over the [getSelectorX()](https://github.com/skydoves/ColorPickerView/blob/master/colorpickerview/src/main/java/com/skydoves/colorpickerview/ColorPickerView.java#L581)?

And I will be grateful for adding simple documentation :)

Reply:
definitely..